scvi/model/base/_base_model.py

```diff
@@ -739,9 +739,6 @@ def __repr__(
         summary_string += "\nTraining status: {}".format(
             "Trained" if self.is_trained_ else "Not Trained"
         )
-        summary_string += "\nLatent model? {}".format(
-            hasattr(self, "latent_data_type") and self.latent_data_type is not None
-        )
```

Comment:
This means models that dont support latent mode yet won't print this... but I wanted them to print it.